docs/reference/search/search-your-data/knn-search.asciidoc

```diff
@@ -563,6 +564,18 @@ minimum similarity for a vector to be considered a match. The `knn` search flow
 * Do not return any vectors that are further away than the configured `similarity`
 --
 
+NOTE: `similarity` is the true <<dense-vector-similarity, similarity>> before it has been transformed into `_score` and boost applied.
```

Comment:
I find this statement a bit confusing. Are we saying the below functions are not the "true" similarity here?

Reply:
I am saying the below functions are how to transform a `_score` back into similarity if you want to change it. 

We take vector similarity and munge it into a score in all sorts of ways.